app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb

```diff
@@ -36,7 +36,7 @@
     margin_bottom: 4,
   } %>
 
-  <p class="govuk-body">If the test result is positive or unclear, you must <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/self-isolation-and-treatment/when-to-self-isolate-and-what-to-do" class="govuk-link">self-isolate</a> for 10 days (the day you took the test is day 0).</p>
+  <p class="govuk-body">If the test result is positive or unclear, you must <a href="https://www.nhs.uk/conditions/coronavirus-covid-19/self-isolation-and-treatment/when-to-self-isolate-and-what-to-do" class="govuk-link">self-isolate</a> for 10 days (the day you took the test is day 0).You can stop self-isolating at the start of day 6 if you get 2 negative rapid lateral flow test results on days 5 and 6 and do not have a temperature. Tests must be at least 24 hours apart. If either test is positive, wait 24 hours before testing again. </p>
```

Comment:
I think we're missing a space between `(the day you took the test is day 0).` and `You can stop self-isolating...`